package/health_daemon/src/gnss_bias_monitor.c

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (C) 2018 Swift Navigation Inc.
+ * Contact: Swift Navigation <dev@swiftnav.com>
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+
+#include <stdlib.h>
+#include <string.h>
+
+#include <libpiksi/logging.h>
+#include <libpiksi/sbp_zmq_pubsub.h>
+
+#include <libsbp/sbp.h>
+#include <libsbp/observation.h>
+
+#include "health_monitor.h"
+#include "utils.h"
+
+#include "gnss_bias_monitor.h"
+
+#define GNSS_BIAS_ALERT_RATE_LIMIT (240000) /* ms */
+
+static health_monitor_t *gnss_bias_monitor;
```

Comment:
Let's add docs to each monitor explaining the goal of the health monitor.

Reply:
Will do. In general docs need to improve on this daemon. WIP https://github.com/swift-nav/firmware_team_planning/issues/447